app/models/membership.rb

```diff
@@ -53,4 +72,17 @@ def user_on_campus
     return if user.on_campus?
     errors.add :user, 'must be living on campus to join a group'
   end
+
+  def update_counter_cache
+    return unless status_changed?
+    increment_counter_cache
```

Comment:
Hmm shouldn't this actually be `if accepted?`? I'm just thinking about cases where it goes from `requested` to `invited` hypothetically - that shouldn't trigger an increment of the counter cache. Thoughts?